src/store/model-store.js

```diff
@@ -49,10 +50,22 @@ class ModelStore extends Store {
     getModelByActionId(actionId) {
         return modelsByActionId[actionId];
     }
+
+
 }
 
 const store = new ModelStore();
 
+function deleteModelByAddress(address){
+    let result = readAddress(address);
+    if(result.type !== 'model' ) {
+        throw new Error('Address is not an model.');
+    }
+    delete models[result.index];
+    let itemsOfModel = ItemStore.getItems().filter(item => item.modelAddress === address);
+    itemsOfModel.forEach(item => ItemStore.deleteItemByAddress(item.address));
```

Comment:
NACK: When did `ItemStore.deleteItemByAddress` become a thing? Just like `ModelStore.deleteModelByAddress` should not exist, this should never have happened either.

Instead, `ItemStore` should be handling the `DELETE_MODEL` action on its own and deleting the items of that model.